.github/workflows/pypi-release.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -U setuptools setuptools_scm wheel twine
     - name: Build and check source tarball
-      run: python setup.py sdist bdist_wheel
+      run: python _old_setup_py.txt sdist bdist_wheel
```

Comment:
oops, this isnt right.

Reply:
this whole action needs a rewrite -- follow the [yt_aspect example](https://github.com/data-exp-lab/yt_aspect/blob/main/.github/workflows/publish-to-pypi.yaml)